apps/explorer/src/components/modals/ViewRawData.tsx

```diff
@@ -0,0 +1,93 @@
+import { Button } from "@/components/Button";
+import { Container } from "@/components/Container";
+import { Column } from "@/components/Layout";
+import { ModalProps, createModal } from "@/components/Modal";
+import { CopyIcon } from "@/icons/CopyIcon";
+import { styled, useTheme } from "@/styled-components";
+import { Text } from "@/components/Typography";
+import { useEffect, useState } from "react";
+
+export type ViewRawDataModalProps = ModalProps & {
+  data: string;
+};
+
+export const ViewRawDataModal = createModal(
+  (modalProps: ViewRawDataModalProps) => {
+    const theme = useTheme();
+    const [isCopied, setIsCopied] = useState(false);
+
+    useEffect(() => {
+      if (!isCopied) return;
+      const timer = setTimeout(() => {
+        setIsCopied(false);
+      }, 1000);
+
+      return () => clearTimeout(timer);
+    }, [isCopied]);
```

Comment:
we can reuse the `useCopyAddress` hook for this, although maybe we should rename the hook to not be about addresses